From bdfccd1366292ff75460b181ed8860b4b7911ee6 Mon Sep 17 00:00:00 2001
From: Marko Katic <dromede.gmail.com>
Date: Thu, 15 Mar 2012 23:46:42 +0100
Subject: [PATCH 1/5] xcb_util.inc: Remove packaging and modify SRC_URI for multiple ${BPN} names

Xcb-util was split into separate packages during the 0.3.6 -> 0.3.8 transition. 
Update xcb_util.inc to reflect that by removing the older packaging.
Libxcbatom, libxcbaux and libxcbevent are merged into libxcbutil.
Libxcbicccm, libxcbimage, libxcbkeysyms and libxcbwm are now separate packages.
Libxcbproperty and libxcbreply were made obsolete with this commit:

http://cgit.freedesktop.org/xcb/util/commit/?id=81c62e7a04213abefa1ea82819887aaaa3c31a74
---
 meta/recipes-graphics/xcb/xcb-util.inc |   16 +---------------
 1 files changed, 1 insertions(+), 15 deletions(-)

diff --git a/meta/recipes-graphics/xcb/xcb-util.inc b/meta/recipes-graphics/xcb/xcb-util.inc
index cd9923d..7dad545 100644
--- a/meta/recipes-graphics/xcb/xcb-util.inc
+++ b/meta/recipes-graphics/xcb/xcb-util.inc
@@ -18,20 +18,6 @@ DEPENDS += "gperf-native"
 
 PARALLEL_MAKE = ""
 
-SRC_URI = "http://xcb.freedesktop.org/dist/xcb-util-${PV}.tar.bz2"
-
-PACKAGES =+ "libxcbatom libxcbaux libxcbevent libxcbicccm \
-             libxcbimage libxcbkeysyms libxcbproperty \
-             libxcbreply libxcbwm"
-
-FILES_libxcbatom = "${libdir}/libXCBAtom.so.*"
-FILES_libxcbaux = "${libdir}/libXCBAux.so.*"
-FILES_libxcbevent = "${libdir}/libXCBEvent.so.*"
-FILES_libxcbicccm = "${libdir}/libXCBICCCM.so.*"
-FILES_libxcbimage = "${libdir}/libXCBImage.so.*"
-FILES_libxcbkeysyms = "${libdir}/libXCBKeysyms.so.*"
-FILES_libxcbproperty = "${libdir}/libXCBProperty.so.*"
-FILES_libxcbreply = "${libdir}/libXCBReply.so.*"
-FILES_libxcbwm = "${libdir}/libXCBWM.so.*"
+SRC_URI = "http://xcb.freedesktop.org/dist/${BPN}-${PV}.tar.bz2"
 
 inherit autotools pkgconfig
-- 
1.7.4.1

